packages/api/src/graphql/client.ts

```diff
@@ -1,9 +1,31 @@
-import { ApolloClient, InMemoryCache } from '@apollo/client';
-import config from '../helper/config';
+import pkg from "@apollo/client";
+const { ApolloClient, InMemoryCache, HttpLink, ApolloLink } = pkg;
+import { setContext } from "@apollo/client/link/context/index.js";
+import { removeTypenameFromVariables } from "@apollo/client/link/remove-typename/index.js";
```

Comment:
Why's the import link looks so weird?

Reply:
That's because of the esm-commonjs issue